lib/season.rb

```diff
@@ -1,2 +1,8 @@
 class Season
+  attr_reader :games
+  
+  def initialize(game)
+    @games = game
+  end
```

Comment:
Not sure if we will make it to the point of refactoring into these classes but we can always delete them later